tools/go-cli/Dockerfile

```diff
@@ -4,17 +4,25 @@ FROM golang:1.6
 RUN apt-get -y update && \
     apt-get -y install zip
 
-# Download and install the godep tool
-RUN echo "Installing the godep tool"
 ENV GOPATH=/
+
+# Download and install tools
+RUN echo "Installing the godep tool"
 RUN go get github.com/tools/godep
 
+RUN echo "Installing the go-bindata tool"
+RUN go get github.com/jteeuwen/go-bindata/...
```

Comment:
Since we use it as a build tool and not a Go programming package, it's not exactly a code dependency.